src/app/util/af-event.cpp

```diff
@@ -65,7 +64,37 @@ struct EmberEventData
 // Globals
 
 #ifdef EMBER_AF_GENERATED_EVENT_CODE
+extern "C" {
+// Stubs for IAS Zone Client Cluster issue #2057
+EmberEventControl emberAfPluginIasZoneClientStateMachineEventControl;
+void emberAfPluginIasZoneClientStateMachineEventHandler(void){};
+
+// Stubs for IAS Zone Server Cluster issue #2058
+EmberEventControl emberAfPluginIasZoneServerManageQueueEventControl;
+void emberAfPluginIasZoneServerManageQueueEventHandler(void){};
+void emberAfIasZoneClusterServerTickCallback(uint8_t endpoint){};
+
+// Stubs for Network related methods
+EmberEventControl emberAfPluginNetworkSteeringFinishSteeringEventControl;
+EmberEventControl emberAfPluginUpdateTcLinkKeyBeginTcLinkKeyUpdateEventControl;
+void emberAfPluginScanDispatchScanNetworkEventHandler(void){};
+void emberAfPluginNetworkSteeringFinishSteeringEventHandler(void){};
+void emberAfPluginUpdateTcLinkKeyBeginTcLinkKeyUpdateEventHandler(void){};
+// The Network Steering plugin defines a method that use emberEventControlSetInactive without
+// passing a pointer. But this method is unused, so let's override the method call for linking
+// to succeed.
+#define emberEventControlSetInactive(x) (void) 0
```

Comment:
Could we just remove the unused `networkEventWrapper` bit from `af-gen-event.h`?  It'll have to be manually done when regenerating the gen files, but various other manual things have to happen there already, right?

Reply:
I actually plan to remove all that in a followup. I think we can just remove the Network steering plugin and all those will be removed.

Reply:
Works for me.  Just need to make sure the followup happens.

Reply:
Got it. Let me remove it first then. See #2923

Reply:
>  It'll have to be manually done when regenerating the gen files, but various other manual things have to happen there already, right?

AFAIK the 4 manual things that needs to be done for those generated file are:
 * Add back the License
 * Run clang-format
 * Update the EMBER_AF_GENERATED_REPORTING_CONFIG_DEFAULTS macro from gen/endpoint_config.h
 * Add #include "debug-printing-test.h" to gen/gen_config.h

Those are already a bit painful. I wish we won't add more !

Reply:
Sigh. It seems like that even if #2923 let us remove some stubs from the PR, `clusterTickWrapper` from `gen/af-gen-event.h` still calls `emberEventControlSetInactive` as well as `emberAfPushEndpointNetworkIndex` and `emberAfPopNetworkIndex`

So I will add a new manual step to the list as you suggested. I still think it worth landing #2923 though.